client/ayon_core/lib/execute.py

```diff
@@ -209,6 +209,15 @@ def run_ayon_launcher_process(*args, **kwargs):
         # - fill more if you find more
         env = clean_envs_for_ayon_process(os.environ)
 
+    if add_sys_paths:
+        pp_set = frozenset(sys.path)
+        new_pythonpath = list(pp_set)
+        pythonpath = env.get("PYTHONPATH") or ""
+        for path in frozenset(pythonpath.split(os.pathsep)):
+            if path and path not in pp_set:
+                new_pythonpath.append(path)
```

Comment:
```suggestion
        paths_lookup = set(sys.path)
        paths = list(sys.path)
        pythonpath = env.get("PYTHONPATH") or ""
        for path in pythonpath.split(os.pathsep):
            if path and path not in paths_lookup:
                paths_lookup.add(path)
                paths.append(path)
```

`paths` in this example is the `new_pythonpath`.